setup.cfg

```diff
@@ -79,7 +79,12 @@ install_requires =
     # Update CustomTTYColoredFormatter to remove
     colorlog>=4.0.2, <5.0
     configupdater>=3.1.1
-    connexion[flask]>=2.10.0
+    # `airflow/www/extensions/init_views` imports `connexion.decorators.validation.RequestBodyValidator`
+    # connexion v3 has refactored the entire module to middleware, see: /spec-first/connexion/issues/1525
+    # Specifically, RequestBodyValidator was removed in: /spec-first/connexion/pull/1595
+    # The usage was added in #30596, seemingly only to override and improve the default error message.
+    # Either revert that change or find another way, preferably without using connexion internals.
```

Comment:
```suggestion
    # Either revert that change or find another way, preferably without using connexion internals.
    # This limit can be removed after https://github.com/apache/airflow/issues/35234 is fixed 
```

Reply:
I created an issue for that in our repo - to keep track of it. This is what we do when we limit upper-binding - we create a "Task" issue to fix the problem when we do - this way we not only see immediately in the code why we limit it, but we also have an issue to track it which easy easy to track and search.